virtualizarr/writers/icechunk.py

```diff
@@ -96,6 +103,7 @@ def write_variables_to_icechunk_group(
             group=group,
             name=name,
             var=var,
+            append_dim=append_dim,
```

Comment:
This also should be added to the `ds.to_zarr` call above too, so that non-virtual variables can be appended.

Reply:
But I don't think we can rely on that doing validation checks for us.